src/lib/services/results-service.ts

```diff
@@ -0,0 +1,123 @@
+import { Receiver } from '../../models/receiver';
+import constants from "../../constants";
+import __logger from "../logger";
+import { PegReceivedData } from "../../models/peg-received-data";
+import * as fs from "fs";
+import TableHelper from "../parsers/tableHelper";
+import { PockyDB } from '../database/db-interfaces';
+import { ResultRow } from '../../models/database';
+const storage = require("@google-cloud/storage");
+
+export interface IResultsService {
+	returnResultsMarkdown(): Promise<string>
+	generateHtml(winners: Receiver[], results: Receiver[], todayString: string): string
+}
+
+export class ResultsService implements IResultsService {
+	database: PockyDB;
+
+	constructor(database: PockyDB){
+		this.database = database;
+	}
+
+	async returnResultsMarkdown(): Promise<string> {
+		const winnersData: ResultRow[] = await this.database.returnWinners();
+		let resultsData: ResultRow[] = await this.database.returnResults();
```

Comment:
Get these simultaneously using `await Promise.all`. Also why is one `const` and one `let`?

Reply:
Actually, why is the results service calling winners at all? Should that not be the purview of the winners service?

It seems as though this results service is taking on more responsibility that it ought to. Getting and formatting both the winners and results, putting each of those into a file, and uploading the file. Maybe these should be separate concerns?

Reply:
I do agree the results service does a lot, but I'm not exactly sure of the best way to split it up.

Reply:
Yeah it's a tricky one. Might require a little thought on paper or something. Will think it over...

Reply:
Know what, this is a good first step, anyway. We can push this through and create another branch to refactor further.

Reply:
sounds good